packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -145,6 +145,61 @@ Generated by [AVA](https://avajs.dev).
       ],
     ]
 
+## upgrade; update noble ICA
+
+> Under "published", the "fastUsdc" node is delegated to Fast USDC.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.fastUsdc',
+        {
+          nobleICA: 'noble1test1',
+          poolAccount: 'agoric1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqp7zqht',
+          settlementAccount: 'agoric1qyqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc09z0g',
+        },
+      ],
+    ]
+
+> feeConfig: 0.01USDC flat, 0.5% variable, 20% contract cut
```

Comment:
the fix for 0.5% variable is coming in #11054 . This part should not have been introduced until there, but it all ends up in the same place...